kernels/ZendEngine2/string.c

```diff
@@ -305,19 +311,25 @@ void zephir_camelize(zval *return_value, const zval *str) {
 }
 
 /**
- * Convert a camelized to a dash/underscored texts
+ * Convert a camelized to a dash/underscored texts (an optional delimiter can be specified)
  */
-void zephir_uncamelize(zval *return_value, const zval *str) {
+void zephir_uncamelize(zval *return_value, const zval *str, const zval *delimiter) {
 
 	unsigned int i;
 	smart_str uncamelize_str = {0};
-	char *marker, ch;
+	char *marker, ch, delim;
 
 	if (Z_TYPE_P(str) != IS_STRING) {
```

Comment:
yeah that part is fine, I would just like some error check like in my last comment,
since when someone would be to lazy to read the source code and passes something
unexpected the function will not do what he wants it to do, without him figuring out,
likely leading to unnecessary issues and wasted time on his part.